test/specs/angularjs-scenario.spec.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+const {expect} = require("chai");
+const {Key} = require("protractor");
+
+const HomePage = require("../utilities/pageObjects/angularJs/home-page");
+const SearchResultPage = require("../utilities/pageObjects/angularJs/"
+                                                         + "search-result-page");
+const TutorialPage = require("../utilities/pageObjects/angularJs/tutorial-page");
+const homePage = new HomePage();
+const tutorialPage = new TutorialPage();
+const searchResultsPage = new SearchResultPage();
+
+
+describe("An AngularJS website scenario", () => {
+    before(() => browser.waitForAngularEnabled(true));
+    after(() => browser.close());
+
+
+    it("should open AngularJS homepage", async () => {
+        await homePage.openPage();
+        const titleOfPage = await homePage.getTitleOfPage();
+        expect(titleOfPage).to.include("AngularJS");
+    });
+
+
+    it("should go to Tutorial page", async () => {
+        await homePage.learnButton.clickOn();
+        await homePage.tutorialLink.clickOn();
+        const tutorialUrl = await homePage.getCurrentUrl();
+        expect(tutorialUrl).to.be.eql("https://docs.angularjs.org/tutorial");
+    });
+
+
+    it("should find an article about \"bind\"", async () => {
+        await tutorialPage.searchField.inputText("bind");
+        await tutorialPage.wait(3000);
+        await tutorialPage.searchField.element.sendKeys(Key.ENTER);
```

Comment:
better to wrap with smth in Element. API methods should not be used on test layer.

Reply:
ok, thanks! I have corrected that part of my code.